models/daily_deal.py

```diff
@@ -1,4 +1,17 @@
 from models.common import WarframeObj
+from collections import defaultdict
+from dateutil.parser import isoparse
 
 class DailyDeal(WarframeObj):
-    pass
\ No newline at end of file
+    def __init__(self, json_daily_deal : dict):
+        json_daily_deal = defaultdict(lambda : None, json_daily_deal)
+        self.sold = json_daily_deal['sold']
+        self.item = json_daily_deal['item']
+        self.unique_name = json_daily_deal['uniqueName']
+        self.total = json_daily_deal['total']
+        self.eta = json_daily_deal['eta']
```

Comment:
this may need to be an isoparse... see what gets returned by the api and if it is of the form like "YYYY.MM.DD.123456789Z" then make it an isoparse